chips/nrf52/src/uicr.rs

```diff
@@ -56,13 +65,67 @@ register_bitfields! [u32,
             /// NFC operation
             NFC = 1
         ]
+    ],
+    /// Enable external circuitry to be supplied from VDD pin
+    ExtSupply [
+        /// Enable external circuitry to be supplied from VDD pin
+        EXTSUPPLY OFFSET(0) NUMBITS(1) [
+            /// No current can be drawn from the VDD pin
+            DISABLED = 0,
+            /// It is allowed to supply external circuitry from the VDD pin
+            ENABLED = 1
+        ]
+    ],
+    /// GPIO reference voltage / external output supply voltage
+    RegOut [
+        /// Output voltage from REG0 regulator stage
+        VOUT OFFSET(0) NUMBITS(3) [
+            V1_8 = 0,
+            V2_1 = 1,
+            V2_4 = 2,
+            V2_7 = 3,
+            V3_0 = 4,
+            V3_3 = 5,
+            DEFAULT = 7
+        ]
     ]
 ];
 
 pub struct Uicr {
     registers: StaticRef<UicrRegisters>,
 }
 
+#[derive(Copy, Clone, PartialEq)]
+/// Output voltage from REG0 regulator stage.
+/// The value is board dependent (e.g. the nRF52840dk board uses 1.8V
+/// whereas the nRF52840-Dongle requires 3.0V to light its LEDs).
+/// When a chip is out of the factory or fully erased, the default value (7)
+/// will output 1.8V.
+pub enum RegOut0 {
```

Comment:
Rename to `Regulator0Output` as to not confuse with "regular output".
```suggestion
pub enum Regulator0Output {
```